.travis.yml

```diff
@@ -23,8 +23,8 @@ addons:
 
 install:
   - export CXX="g++-4.8"
-  - npm install -g npm@latest
-  - npm install
+  - npm install -g yarnpkg
```

Comment:
According to the alt install methods listed on the docs site https://yarnpkg.com/en/docs/install

It looks like `npm i -g yarn` is the one we want here.